cypresse2e/cypress/integration/e2eTests/joinNow/joinNow.spec.js

```diff
@@ -41,6 +41,48 @@ describe('FightPandemics Sign Up Page', () => {
 			validEmailRequired.contains('small', errorMessages.invalidEmail);
 		});
 
+		it('Entering @ twice or more in email triggers error', () => {
+			var emailField = joinNow.getEmailField();
+			emailField.should('be.visible').and('have.attr', 'name', 'email');
+			emailField.type(emailAndPassword.invalidSampleEmail2).focus().blur();
```

Comment:
@JayashreeSamantasinghar a function for emailField() was created in this file, so you do not need to write out the whole code: 
```
var emailField = joinNow.getEmailField();
emailField.should('be.visible').and('have.attr', 'name', 'email');
emailField.type(emailAndPassword.invalidSampleEmail2).focus().blur();
```

it can be refactored to:

```
emailField().type(emailAndPassword.invalidSampleEmail2).focus().blur();
```

Reply:
@jessica-lau 
I made the changes suggested by you, please review it